lib/commands/start.js

```diff
@@ -59,8 +60,20 @@ class StartCommand extends Command {
                     });
                 }).then(() => {
                     if (!argv.quiet) {
+                        const isInstall = argv._ && argv._.length && argv._[0] === 'install';
```

Comment:
@acburdine is there a better way to detect the original command that was used here? I couldn't find a better way, but maybe you know one.

Reply:
hmm - you could check `process.argv[2]` - based on how ghost-cli starts up a value is guaranteed to exist there.

Reply:
Worked like a charm!